libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/NavigationEventDispatcherTest.java

```diff
@@ -77,14 +75,9 @@ public void setup() throws IOException {
     String body = loadJsonFixture(PRECISION_6);
     DirectionsResponse response = gson.fromJson(body, DirectionsResponse.class);
     route = response.routes().get(0);
-    routeProgress = RouteProgress.builder()
-      .stepDistanceRemaining(100)
-      .legDistanceRemaining(100)
-      .distanceRemaining(100)
-      .directionsRoute(route)
-      .stepIndex(0)
-      .legIndex(0)
-      .build();
+
+    routeProgress = buildRouteProgress(route, 100, 100,
+      100, 0, 0);
   }
 
```

Comment:
Generally is preferable having only 1 assertion per test 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=57 The same "rule" applies to _Behavior tests_ (`verify`).
This applies to other tests included in `NavigationEventDispatcherTest`.